webapp/src/Twig/TwigExtension.php

```diff
@@ -1406,4 +1408,35 @@ public function extensionToMime(string $extension): string
     {
         return DOMJudgeService::EXTENSION_TO_MIMETYPE[$extension];
     }
+
+    public function domjudgeMarkdownToHTML(string $markdown): string
+    {
+        $latexFound = [];
+        while (true) {
```

Comment:
It's not very clear to me what you are doing here? Are you making sure that we aren't markdowning any of the LaTex code? If so, maybe at least explain that we are doing that.